modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/RTImage.java

```diff
@@ -115,6 +115,10 @@ private RTTexture getTexture() {
                     (int) Math.ceil(width * pixelScale),
                     (int) Math.ceil(height * pixelScale),
                     Texture.WrapMode.CLAMP_NOT_NEEDED);
+            if (txt == null) {
+                log.fine("RTImage::getTexture : return null because rt texture not allocated");
+                return null;
```

Comment:
If `RTImage.getTexture()` can return `null`, then L147 should probably also check for `null` before calling `readPixels()`.